### PR TITLE
feat(sass): add conditional sass deps for webpack

### PR DESCRIPTION
**Pull request by MartoYankov**
Add sass dependency only if the dev-sass plugin is already installed.

### CODE REVIEW

File: dependencyManager.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ function getRequiredDeps(packageJson) {
         Object.assign(deps, { "awesome-typescript-loader": "~3.1.3" });
     }
 
+    if (isSass({packageJson})) {
```
Comment (vchimev) [inline dependencyManager.js]: Don't we need also `node-sass`?

**Comment by vchimev** [comment]
js

**Comment by vchimev** [comment]
ts

File: dependencyManager.js (hunk 1 above)
Comment (MartoYankov) [reply dependencyManager.js]: No. It turns out node-sass is a dep to sass-loader. :)

**Pull request merged**